lua/session_manager/config.lua

```diff
@@ -9,10 +9,32 @@ local config = {
   }),
 }
 
+--- Replaces symbols into separators and colons to transform filename into a session directory.
+---@param filename string: Filename with expressions to replace.
+---@return string: Session directory
+function config.delimited_session_filename_to_dir(filename)
+  local dir = filename
+  dir = dir:gsub(config.colon_replacer, ':')
+  dir = dir:gsub(config.path_replacer, Path.path.sep)
+  return dir
+end
+
+--- Replaces separators and colons into special symbols to transform session directory into a filename.
+---@param dir string: Path to session directory.
+---@return string: Session filename.
+function config.dir_to_delimited_session_filename(dir)
+  local filename = dir
+  filename = filename:gsub(':', config.colon_replacer)
+  filename = filename:gsub(Path.path.sep, config.path_replacer)
+  return filename
+end
+
 config.defaults = {
   sessions_dir = Path:new(vim.fn.stdpath('data'), 'sessions'),
   path_replacer = '__',
   colon_replacer = '++',
```

Comment:
I would turn this things just into local variables. I don't think that someone will need to configure it it.
```suggestion
  path_replacer = '__',
  colon_replacer = '++',
```

Reply:
I'm a fan of this too